api/src/main/java/keywhiz/api/model/SecretContent.java

```diff
@@ -32,17 +32,18 @@
  */
 @AutoValue
 public abstract class SecretContent {
-  public static SecretContent of(long id, long secretSeriesId, String encryptedContent, ApiDate createdAt,
+  public static SecretContent of(long id, long secretSeriesId, String encryptedContent, String hmac, ApiDate createdAt,
```

Comment:
We set the migration to have a default of empty string, so hmac should always be a string (though it may be empty)